src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -261,6 +261,15 @@
 	<data name="SpineHomeButton_ToolTip_Home" xml:space="preserve">
 		<value>Strona główna</value>
 	</data>
+    <data name="LoadoutGridView__View_Files" xml:space="preserve">
+        <value>Zobacz Pliki Moda</value>
+    </data>
+    <data name="FileTreeNodeView__View" xml:space="preserve">
+        <value>Zobacz</value>
+    </data>
+    <data name="ViewModInfoPage_Title" xml:space="preserve">
+        <value>Informacje Moda</value>
```

Comment:
It should be `Informacje modyfikacji` or optionally `Informacje moda` if the text is too long.
```suggestion
        <value>Informacje modyfikacji</value>
```

Reply:
Yeah, that's the problem. I wrote it as 'Informacje moda' because this text goes in the tab title, same with https://github.com/Nexus-Mods/NexusMods.App/pull/1023#discussion_r1518426680

Our tab titles can't display very many characters (the text gets trimmed very quickly), so I tried to keep it short.

Reply:
I think just change the `"M"` to `"m"` and it will be fine.

```suggestion
        <value>Informacje moda</value>
```

Reply:
Yeah that's fine